src/components/accordion/accordion.ts

```diff
@@ -36,6 +43,11 @@ export class Accordion extends LitElement {
 
   override willUpdate() {

```

Comment:
In Lit `update` lifecycle functions, I recommend using the `changedProperties` argument, and wrapping anything within in an `if` statement, so it only runs when the related props change.